js/spotify.js

```diff
@@ -1,12 +1,56 @@
 // API Docs at: 
 // https://developer.spotify.com/web-api/search-item/
 
+// ## Basic Requirements:
+//  - Enter a keyword to seach for.
+//  - Select "artist" or "track" as the search type.
+//  - Have the names of all search results print as a list.
 
-function searchByArtist(keyword) {
-  var url = 'https://api.spotify.com/v1/search?q='+keyword+'&type=artist';
+//create variables from index.html ids
+ var $form = $("search")
+ var $searchByArtistOrTrack = $("search-type")
+ var $results = $("results")
+ var $input = $("search-keyword")
+
+//search by artist
+$form.on("submit", function searchByArtist(keyword) {
```

Comment:
you can check the selected option with `$searchByArtistOrTrack.find(":selected").val()`